src/i18n/de/nav.ts

```diff
@@ -1,94 +1,81 @@
 import { NavDictionary } from '../translation-checkers';
 
 export default NavDictionary({
-	// Start Here
-	startHere: 'Beginne hier',
+	welcomeWorld: 'Willkommen, Welt!',
 	'getting-started': 'Erste Schritte',
-	install: 'Installation',
-	'editor-setup': 'Editor-Einrichtung',
-	'upgrade-astro': 'Upgrade Astro',
+	'install-and-setup': 'Installation und Konfiguration',
+	'guides/deploy': 'Deine Website einrichten',
```

Comment:
```suggestion
	'guides/deploy': 'Deine Website veröffentlichen',
```
Siehe bereits übersetzte Seite, hier geht es eher um die Veröffentlichung bei einem Hosting-Provider.